h2o-r/h2o-package/render-h2o.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+### Render h2o R package from h2o-3 as a standalone R package h2o
+# script will download recent nightly h2o.jar so you should always rebase your h2o package to recent master
```

Comment:
Technically a rebase is needed if they are using top of tree. However, a user could just take the jar and launch it, right?